src/OagBundle/Resources/views/Default/index.html.twig

```diff
@@ -16,96 +18,134 @@
+                    </li>
+                {% endfor %}
+                <div class="upload-form">
+                    <h3>Import a new source document</h3>
+                    <div>The imported document will be automatically processed with CoVE and added to the IATI document list above.</div>
+                    <div>Currently only XML is supported.</div>
+                    <div>
+                        {{ form_start(source_upload_form) }}
+                        {{ form_label(source_upload_form.documentName) }}
+                        {{ form_errors(source_upload_form.documentName) }}
+                        {{ form_widget(source_upload_form.documentName) }}
+
+                        {{ form_widget(source_upload_form.fileType, { 'attr': {'class': 'hidden-row'} }) }}
+                        {{ form_end(source_upload_form) }}
+                    </div>
+                </div>
+            </ul>
+
         </div>
-    </ul>
 
-    <h2>Enhancement Documents <small><small>[activities/locations]</small></small></h2>
+    </section>
 
-    <ul>
-        {% for enhancingDoc in enhancingDocs %}
-            <li>
-                {{ include('OagBundle::Default/enhancementFile.html.twig', { 'enhancingDoc': enhancingDoc }) }}
-            </li>
-        {% endfor %}
-    </ul>
+    <section>
 
-    <h2>Todo Add</h2>
-    <ul><li>Delete button</li></ul>
+        <div class="content">
+
+            <h2>Enhancement Documents <small>[activities/locations]</small></h2>
+
+            <ul>
+                {% for enhancingDoc in enhancingDocs %}
+                    <li>
+                        {{ include('OagBundle::Default/enhancementFile.html.twig', { 'enhancingDoc': enhancingDoc }) }}
+                    </li>
+                {% endfor %}
+            </ul>
+
+        </div>
+
+    </section>
 
     {% else %}
 
-      <h2>Using the Open Ag Toolbox you can:</h2>
+    <section>
+
+        <div class="content">
+
+            <div class="intro">
+                <h1 class="dark">Using the {{ toolbox_title }} you can...</h1>
+
+                <h2><span class="dark">View</span> how your data will look to the world.</h2>
 
-      <h3>View - How your data will look to the world.</h3>
+                <h2><span class="dark">Check</span> what your data can be used for and improve it.</h2>
 
-      <h3>Check - What your data can be used for and improve it.</h3>
+                <h2><span class="dark">Publish</span> your improved activity file.</h2>
+            </div>
+
+            <h1 class="dark">Get Started</h1>
+
+            <div class="upload-form">
+                <h3>Import a new source document</h3>
+                <hr>
+                <div>The imported document will be automatically processed with CoVE and added to the IATI document list above.</div>
+                <div>Currently only XML is supported.</div>
+                <div>
+                    {{ form_start(source_upload_form) }}
```

Comment:
Is this repeated from the earlier form code on line 61? Can we use the fileUploadWidget or something similar to reduce repetition?